src/App.js

```diff
@@ -65,9 +67,13 @@ const App = compose(
             forceSelectRole({
               api,
               isLoadingUser,
-              Component: ({ match, ...props }) => (
-                <ArrangerDashboard basename={match.url} {...props} />
-              ),
+              Component: ({ match, ...props }) => {
+                const validatedContent = validateJWT({ jwt: state.loggedInUserToken });
+                if (!get(validatedContent, 'context.user.roles', []).includes('ADMIN')) {
+                  history.replace('/dashboard');
```

Comment:
similar to forceSelectRole, could we use a `<Redirect to="..." />` rather than modifying url directly?
```
return !get(...).includes('ADMIN') ? <Redirect to="..." /> : <ArrangerDashboard ... />
```